Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -142,8 +142,14 @@ namespace Multiplayer
         }
 
         // Start the configured server if it's available
+        AZStd::string projectPath(AZ::Utils::GetProjectPath().c_str());
```

Comment:
FYI the AZ::IO::Path/AZ::IO::FixedMaxPath classes have a `.c_str()` function on it.
Also it has a `.Native()` function that returns a non-const lvalue reference to the path storage type.

Reply:
This could be the following
```
AZ::IO::FixedMaxPath projectPath(AZ::Utils::ProjectPath());
processLaunchInfo.m_commandlineParameters = AZStd::string::format(
    R"("%s" --project-path "%s" --editorsv_isDedicated true --sv_defaultPlayerSpawnAsset "%s")",
    serverPath.c_str(),
    projectPath.c_str(),
    static_cast<AZ::CVarFixedString>(sv_defaultPlayerSpawnAsset).c_str());